sdk/communication/azure-communication-common/CHANGELOG.md

```diff
@@ -1,11 +1,13 @@
 # Release History
 
-## 1.3.0-beta.2 (Unreleased)
+## 2.0.0-beta.1 (Unreleased)
 
 ### Features Added
 
 ### Breaking Changes
 
+- Added support for a new communication identifier `MicrosoftBotIdentifier`.
```

Comment:
should be it under the added features?

Reply:
I'd go with both...see my comment in the js sdk

Reply:
Updated